React/Base/RCTImageSource.m

```diff
@@ -57,13 +59,31 @@ + (RCTImageSource *)RCTImageSource:(id)json
   }
 
   NSURL *imageURL;
+  NSDictionary *headers = nil;
   CGSize size = CGSizeZero;
   CGFloat scale = 1.0;
   BOOL packagerAsset = NO;
   if ([json isKindOfClass:[NSDictionary class]]) {
     if (!(imageURL = [self NSURL:RCTNilIfNull(json[@"uri"])])) {
       return nil;
     }
+    headers = [self NSDictionary:RCTNilIfNull(json[@"headers"])];
+#if RCT_DEBUG
```

Comment:
Is it okay to call RCTLogError in production or do I need to keep that part guarded by a `#if RCT_DEBUG`?